cdap-client-tests/src/main/java/co/cask/cdap/client/app/FakeApp.java

```diff
@@ -55,7 +58,10 @@ public void configure() {
     createDataset(DS_NAME, FakeDataset.class.getName());
     addProcedure(new FakeProcedure());
     addFlow(new FakeFlow());
-    addSpark(new FakeSpark());
+    addWorkflow(new FakeWorkflow());
+    scheduleWorkflow(new TimeSchedule(SCHEDULE_NAME, "", "0 0 1 1 *"), FakeWorkflow.NAME);
```

Comment:
In that line, and the next, use ``Schedules.create...`` instead of instantiating the ``TimeSchedule`` and ``StreamSizeSchedule`` classes directly. They are in the ``api.internal`` package, and are not meant to be used directly by app developers.